README.md

```diff
@@ -1,19 +1,87 @@
 # Submit a GAR Public Site UI
 
-## Docker Usage
-The following depends on a .env file in the root of this project with the required environment variables. This will start a local test instance of keycloak for local testing.
+## Requirements
+- docker
+- docker-compose
+- node
+- npm
+- your email added to GOV Notify Dev whitelist by repository administrator
+
+## Development
+
+The following depends on a `.env.dev` file in the root of this project with
```

Comment:
Something worth mentioning is sourcing .env.dev won't work in the mac environment as it doesn't have the export command as needed by mac. Unless the administrator gives it with export command.

Reply:
I have replaced the `source` command with `for` loop, covering both MacOS and POSIX Linux